build.sbt

```diff
@@ -194,7 +194,7 @@ val defaultMultiJvmOptions: List[String] = {
     case key if knownPrefix.exists(pre => key.startsWith(pre)) => "-D" + key + "=" + System.getProperty(key)
   }
 
-  "-Xmx128m" :: properties
+  "-Xmx256m" :: properties
```

Comment:
Akka is using 256 MB when running multi-jvm tests. 128 is too low and I think it may have some impact on our tests stability.

Reply:
It hasn't been an issue locally and using a more constrained setting helps to reproduce a poor VM on local laptops. In CI VMs, running 3x JVM's constrained to 256m seems likely to make the constrain be the mem on the VM and not the setting so it becomes harder to tell how much mem is actually being used. Running beefier forked JVMs can even be counterproductive in limited VM environments since Cassandra and Kafka will have stronger competitors when allocating mem.


I think the actual constraint causing instability in tests is differences in CPU power.

I think forcing 128m locally helps local envs behave more like a limited CI env. Even though we're not constraining memory allocation for Cassandra nor Kafka, and neithert we are constraining the CPU usage on any of the forked processes.

Reply:
So, if the constraint that is causing instability is CPU power, limiting the memory locally to make it behave more like a limited CI env won't help much, right? I mean, we won't be limiting where the constraint is. Anyway, having a little more memory can reduce the need for GC runs, which will reduce CPU pressure as well. But I'm not sure how much this will be relevant considering the tests aren't (or at least are not supposed to be) long-living processes.